src/assets/wise5/authoringTool/edit-branch/edit-branch.component.ts

```diff
@@ -271,4 +259,29 @@ export class EditBranchComponent extends AbstractBranchAuthoringComponent {
       this.node.transitionLogic.whenToChoosePath = 'studentDataChanged';
     }
   }
+
+  private removePaths(branchPaths: any[], params: any): void {
```

Comment:
Type CreateBranchParams? Maybe we want to rename CreateBranchParams to EditBranchParams?
Do we need to pass in branchPaths, or can we get it via ```this.branchPaths```?
```suggestion
  private removePaths(params: CreateBranchParams): void {
```